susemanager-utils/susemanager-sls/susemanager-sls.changes

```diff
@@ -1,3 +1,4 @@
+- Add missing symlink to generate "certs" for SLE Micro 5.0 (bsc#1188503)
```

Comment:
It's not to generate certs, but to install them, right?

Reply:
to generate the "certs" state for SLE Micro... I'm fixing this